admin-site/src/OccasionsList.svelte

```diff
@@ -110,14 +120,28 @@
   emptyArrayMessage = "This happens on occasion. No occasions for this event yet.">
 
   {#each  dateSortedOccasions as item (item.id)}
-    <Button on:click={() => occasionButton(item.id)}
-    buttonHtml = '<h3 class="m-0">{item.label}  - {item.label} </h3> <h5>{item.locationCity} - {moment(item.startDateTime).format("LL")} - id:{item.id}</h5>'
-    value = {item.id}/> 
+    {#if item.locationCity == null}
+      <Button on:click={() => occasionButton(item.id)}
+        buttonHtml = '<h3 class="m-0">{item.label} - {item.label}</h3> <h5>City Name - {moment(item.startDateTime).format("LL")} - id:{item.id}</h5>'
+        value = {item.id}/>
+    {:else}
+      <Button on:click={() => occasionButton(item.id)}
+        buttonHtml = '<h3 class="m-0">{item.label} - {item.label}</h3> <h5>{item.locationCity} - {moment(item.startDateTime).format("LL")} - id:{item.id}</h5>'
+        value = {item.id}/> 
+    {/if} 
   {/each}
   
 </Array>
+<hr>
+<!-- Block button, more consistent with other button UI -->
+<Button on:click={openForm}
+    buttonText = "Occasion Creation Form"
```

Comment:
Let's use 'Create a new occasion' for this message (and same format for events)

Reply:
Yep, that's a better choice. I've also sentence cased all the buttons.